server/src/repositories/shared-link.repository.ts

```diff
@@ -114,7 +114,7 @@ export class SharedLinkRepository {
             .as('assets'),
         (join) => join.onTrue(),
       )
-      .select((eb) => eb.fn.toJson('assets').as('assets'))
+      .select('assets.assets')
```

Comment:
Did you test/validate that this works fine?

Reply:
Yes. It's from the other PR. The current approach looks like `{ assets: { assets: AssetEntity[] } }`